system/usbmsc/usbmsc_main.c

```diff
@@ -428,6 +491,41 @@ int main(int argc, FAR char *argv[])
       return EXIT_FAILURE;
     }
 
+  /* Add const LUNs */
+
+  if (CONFIG_SYSTEM_USBMSC_DEVPATH1[0] != '\0')
```

Comment:
should we check CONFIG_SYSTEM_USBMSC_DEVPATH1 < 0 instead

Reply:
> should we check CONFIG_SYSTEM_USBMSC_DEVPATH1 < 0 instead

But the type of `CONFIG_SYSTEM_USBMSC_DEVPATH1` is string not int.
Do you mean adding another new config?

Reply:
sorry, I mean SYSTEM_USBMSC_DEVMINOR1.

Reply:
> sorry, I mean SYSTEM_USBMSC_DEVMINOR1.

Get, done.